packages/web-components/src/accordion/accordion-item/accordion-item.styles.ts

```diff
@@ -0,0 +1,236 @@
+import { css } from '@microsoft/fast-element';
+import {
+  borderRadiusMedium,
+  borderRadiusSmall,
+  colorNeutralForeground1,
+  colorNeutralForegroundDisabled,
+  colorStrokeFocus1,
+  colorStrokeFocus2,
+  colorTransparentBackground,
+  fontFamilyBase,
+  fontSizeBase200,
+  fontSizeBase300,
+  fontSizeBase400,
+  fontSizeBase500,
+  fontWeightRegular,
+  lineHeightBase200,
+  lineHeightBase300,
+  lineHeightBase400,
+  lineHeightBase500,
+  spacingHorizontalM,
+  spacingHorizontalMNudge,
+  spacingHorizontalS,
+} from '../../theme/design-tokens.js';
+
+export const styles = css`
+  /* --- Default styles --- */
+
+  :host {
+    max-width: fit-content;
+  }
+
+  :host .heading {
+    height: 44px;
+    display: grid;
+    position: relative;
+    vertical-align: middle;
+    padding-left: ${spacingHorizontalMNudge};
+    padding-right: ${spacingHorizontalM};
+    border-radius: ${borderRadiusMedium};
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
+    font-weight: ${fontWeightRegular};
+    line-height: ${lineHeightBase300};
+    grid-template-columns: 28px 28px auto 28px;
+  }
+
+  :host .heading .heading-content {
+    height: 100%;
+    display: flex;
+    align-items: center;
+  }
+
+  :host .button {
+    box-sizing: border-box;
+    appearance: none;
+    border: none;
+    outline: none;
+    text-align: left;
+    cursor: pointer;
+    font-family: inherit;
+    height: 44px;
+    color: ${colorNeutralForeground1};
+    background: ${colorTransparentBackground};
+    line-height: ${lineHeightBase300};
+    height: auto;
+    padding: 0;
+    font-size: inherit;
+    grid-column: auto / span 2;
+    grid-row: 1;
+  }
+
+  :host .button::before {
+    content: '';
+    position: absolute;
+    inset: 0px;
+    cursor: pointer;
+    border-radius: ${borderRadiusSmall};
+  }
+
+  :host .icon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    pointer-events: none;
+    position: relative;
+    height: 100%;
+    padding-right: ${spacingHorizontalS};
+    grid-column: 1 / span 1;
+    grid-row: 1;
+  }
+
+  :host .region {
+    display: none;
+    margin-left: ${spacingHorizontalM};
+    margin-right: ${spacingHorizontalM};
+  }
+
+  :host .heading ::slotted(*) {
+    display: flex;
+    align-self: center;
+  }
+
+  :host slot[name='collapsed-icon'],
+  :host slot[name='expanded-icon'] {
+    display: flex;
+    align-items: center;
+  }
+
+  :host slot[name='expanded-icon'] {
+    display: none;
+  }
+
+  :host .heading ::slotted(span[slot='start']),
+  :host .heading ::slotted(span[slot='end']) {
```

Comment:
With this scenario as well, I don't think slotted is dependent on either of the preceding selectors.